dockstore-webservice/src/main/java/io/openapi/api/impl/ServiceInfoApiServiceImpl.java

```diff
@@ -0,0 +1,78 @@
+package io.openapi.api.impl;
+
+import java.util.Date;
+import java.util.Optional;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
+
+import io.dockstore.webservice.core.User;
+import io.openapi.api.ServiceInfoApiService;
+import io.openapi.model.Service;
+import io.openapi.model.ServiceType;
+
+public class ServiceInfoApiServiceImpl extends ServiceInfoApiService {
+    @Override
+    public Response getServiceInfo(SecurityContext securityContext, ContainerRequestContext value, Optional<User> user) {
+        Service service = getService();
+        return Response.ok().entity(service).build();
+    }
+
+    public static Service getService() {
+        Service service = new Service();
+        service.setId("1");
+        service.setName("Dockstore");
+        service.setType(getServiceType());
+        service.setDescription("The GA4GH Tool Registry API is a standard for listing and describing available tools (both stand-alone, Docker-based tools as well as workflows in CWL, WDL, Nextflow, or Galaxy) in a given registry,");
```

Comment:
Should this mention Dockstore? I could see it other way; as other fields mention it. Depends if this describes the API or the implementation.

Reply:
Yes, it from https://github.com/ga4gh-discovery/ga4gh-service-info/blob/develop/service-info.yaml#L57 I think it is reasonable to say something like "Dockstore is an implementation of the GA4GH Tool Registry API ..."